ecommerce/static/js/views/coupon_form_view.js

```diff
@@ -372,15 +372,15 @@ define([
                 var isBenefitPercentage = this.$('[name=benefit_type]:checked').val() === 'Percentage',
                     maxValue = isBenefitPercentage ? '100' : '';
 
-                this.setLimitToElement(this.$('[name=benefit_value]'), maxValue, 0);
+                this.setLimitToElement(this.$('[name=benefit_value]'), maxValue, 1);
```

Comment:
Wow, we had to override this entire huge file just to get this? I'm really wondering if there's a better way to change this value, e.g. through some setting. But since Hawthorn's already here that may not work out this time around.